install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -553,6 +553,9 @@ data:
 {{- if .Values.svcSourceRangeCheck }}
   enable-svc-source-range-check: {{ .Values.svcSourceRangeCheck | quote }}
 {{- end }}
+{{- if .Values.enableClusterIPExternalAccess }}
+  enable-cluster-ip-external-access: {{ .Values.enableClusterIPExternalAccess | quote }}
```

Comment:
shouldn't this have bpf-lb- prefix?

Reply:
I put this under `bpf.externalClusterIP` which is more inline with the other options.